### PR TITLE
Middleware

### PR DESCRIPTION
**Pull request by Riceqrisp**

### CODE REVIEW

File: GiTinder/Controllers/UsersMockController.cs [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+//            }
+//            else
+//            {
+//                Response.StatusCode = 403;
+//                responseBody = new ErrorResponseBody("Unauthorized request!");
+//            }
+//            return responseBody;
+//        }
+
+//    }
```
Comment (adamgyulavari) [inline GiTinder/Controllers/UsersMockController.cs]: @Riceqrisp delete the file

File: GiTinder/Models/Profile.cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+ 
+        public Profile(string usertoken)
+        {
+            this.UserToken = usertoken;
+          
+        }
+
+    }
+}
```
Comment (adamgyulavari) [inline GiTinder/Models/Profile.cs]: @Riceqrisp delete this

File: GiTinder.Tests/Controllers/UserControllerTests.cs [hunk 3]
```diff
@@ -23,6 +23,8 @@ public class UsersControllerTests
         Mock<HttpContext> httpContext;
         Mock<ProfileServices> profileServices;
 
+        public object SimpleRepository { get; private set; }
```
Comment (adamgyulavari) [inline GiTinder.Tests/Controllers/UserControllerTests.cs]: @Riceqrisp remove this

File: GiTinder.Tests/Controllers/LoginRequestBodyFactory.cs [hunk 4]
```diff
@@ -16,6 +16,15 @@ public static LoginRequestBody CreateLoginRequestBodyWithNullUsername()
...
         }
 
+        public static LoginRequestBody CreateLogoutRequestBodyWithNullUsertoken()
+        {
+            return new LoginRequestBody()
+            {
+                Username = "",
+                AccessToken = "X-Gitinder-Token"
+            };
+        }
```
Comment (adamgyulavari) [inline GiTinder.Tests/Controllers/LoginRequestBodyFactory.cs]: @Riceqrisp remove this method after checking if its used

**Review by adamgyulavari — CHANGES_REQUESTED** [review]

File: GiTinder/Controllers/UsersMockController.cs (hunk 1 above)
Comment (Riceqrisp) [reply GiTinder/Controllers/UsersMockController.cs]: deleted

File: GiTinder/Models/Profile.cs (hunk 2 above)
Comment (Riceqrisp) [reply GiTinder/Models/Profile.cs]: deleted

File: GiTinder.Tests/Controllers/UserControllerTests.cs (hunk 3 above)
Comment (Riceqrisp) [reply GiTinder.Tests/Controllers/UserControllerTests.cs]: removed.

File: GiTinder.Tests/Controllers/LoginRequestBodyFactory.cs (hunk 4 above)
Comment (Riceqrisp) [reply GiTinder.Tests/Controllers/LoginRequestBodyFactory.cs]: removed .

**Review by adamgyulavari — APPROVED** [review]

**Pull request merged**